services/src/users/user.rs

```diff
@@ -6,15 +6,17 @@ use uuid::Uuid;
 use crate::error::{Error, Result};
 use crate::error;
 
-pub trait UserInput {
+// TODO: move elsewhere
```

Comment:
why? do we not want to have todos in the code?

Reply:
TODOs are okay, that is why I did not criticize the other ones.

This one looks like something you would either rather do via refactoring at some point automatically or will be there forever because there is no necessity to do it feature-wise.

Reply:
389c8ef7d1974f159120c425a7aa3c627809330d